pkg/remediator/remediator.go

```diff
@@ -82,6 +86,8 @@ type Interface interface {
 	ManagementConflict() bool
 	// ConflictErrors returns the errors the remediator encounters.
 	ConflictErrors() []status.ManagementConflictError
+	// FightErrors returns the fight errors (KNV2005) the remediator encounters.
+	FightErrors() map[core.ID]status.Error
```

Comment:
Should we maybe move this method to be on the FightHandler?
Then we could expose the FightHandler as a public field and call it from there. It might make it cleaner to move the locks out of the remediator. 

We could also do the same with a ConflictHandler.

Reply:
Oh, I see you already did it this way, nice!

Never mind then.